climpred/classes.py

```diff
@@ -616,6 +626,10 @@ def verify(
             comparison=comparison,
             **metric_kwargs,
         )
+        if isinstance(self._temporally_smoothed, dict):
```

Comment:
You can just do `if self._temporally_smoothed:` here. If it's an empty dict it evaluates False. If it is None, it evaluates False.

I would also put into `PredictionEnsemble` init `self._temporally_smoothed = None` to instantiate it there and hint that it can be modified elsewhere.